.env.app

```diff
@@ -1,6 +1,9 @@
 # feature flags
 REACT_APP_FEATURE_PENDO=true
 
+# mixpanel
+REACT_APP_FEATURE_MIXPANEL=9d304465fc72224aead9e027e7c24356
```

Comment:
Add API_KEY suffix terminology for consistency?

Reply:
That’s actually totally wrong. The key is token I’ll update.